requirements.txt

```diff
@@ -35,3 +35,5 @@ six==1.10.0
 urllib3==1.21.1
 Werkzeug==0.12.2
 wrapt==1.10.10
+elasticsearch-dsl>=5.0.0,<6.0.0
```

Comment:
the last thing,
let's use forked version
git+https://github.com/bloogrox/elasticsearch-dsl-py@boost-mode